docs/getting-started.md

```diff
@@ -80,99 +84,285 @@ await Kilt.init({ address: YOUR_CHAIN_ADDRESS })
 Again, this is asynchronous, so be sure to wrap this in an `async` function as described above.
 Add this line to the `async` functions wrapping the examples below if you run them individually.
 
-## 2. How to generate an Identity
+### 1.2. Connect to a Chain
 
-To generate an Identity first you have to generate a [BIP39 mnemonic](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) and then use it to create the Identity:
+Either explicitly:
+
+```typescript
+Kilt.config({ address: YOUR_CHAIN_ADDRESS })
+await Kilt.connect()
+```
+
+Or by setting a default address in the configuration, connecting implicitly.
+
+```typescript
+Kilt.config({ address: YOUR_CHAIN_ADDRESS })
+```
+
+Note that calling (as described in [1.2](#initializing-the-kilt-sdk))
+
+```typescript
+Kilt.init({ address: YOUR_CHAIN_ADDRESS })
+```
+
+initializes the SDK _and_ sets the config, so it is related to the second approach.
+
+There are of KILT chains which you can use, each one having a different address:
+
+1. A local node: `ws://127.0.0.1:9944`
+2. The prod-net: `wss://peregrine.kilt.io`
```

Comment:
Maybe we should rename this to something else?

Reply:
After our discussion, I have renamed it to `test-net`.